spec/controllers/unsubmitted_courses_controller_spec.rb

```diff
@@ -18,5 +18,19 @@
       expect(response.body).to_not have_content(course.title)
       expect(response.body).to have_content(course2.title)
     end
+
+    it 'should show course creation date' do
+      course = create(:course, title: 'My awesome course',
```

Comment:
To DRY this test out some, you can prepare the common data outside of the individual 'it' blocks, by using `let`. You can find examples of this in many other tests.